src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -86,6 +87,21 @@ private static Person createEditedPerson(Person personToEdit, EditPersonDescript
         return new Person(updatedName, updatedPhone, updatedEmail, updatedAddress, updatedNote, updatedTags);
     }
 
+    /**
+     * Gets the warning message for the command's execution.
+     *
+     * @return The warning message, or an empty string if none.
+     */
+    private String getMessageWarn(Person editedPerson) {
+        boolean isPhoneOfExpectedFormat = editedPerson.getPhone().isExpectedFormat();
+
+        if (!isPhoneOfExpectedFormat) {
+            return String.format(MESSAGE_WARN, Phone.MESSAGE_EXPECTED);
+        }
+
+        return "";
+    }
```

Comment:
Same as `AddCommand`

Reply:
maybe u can do this in EditCommandMessages class and for the others as well? i abstracted out all the messages into their respective command classes @dillontkh @jjchee77

Reply:
> maybe u can do this in EditCommandMessages class and for the others as well? i abstracted out all the messages into their respective command classes @dillontkh @jjchee77

I have moved the warning message template to the `Messages.java` class, I don't see message classes specific to each command though. Thanks for the suggestion! However, I'm not sure if we should move all user visible messages to a single class rather than contain it in it's own respective class, would make sense if the messages are shared across multiple classes.